tests/integration/helpers.py

```diff
@@ -4,6 +4,18 @@
 
 CHARMS_PATH = pathlib.Path(__file__).parent / 'charms'
 
+# Test SSH key pair - only for use in integration tests
+TEST_SSH_PRIVATE_KEY = """-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW
+QyNTUxOQAAACDBApPszf6ki9VNAWpAsU6VY1iM59qe5HyE08zr8tw0SgAAAKDNXMCLzVzA
+iwAAAAtzc2gtZWQyNTUxOQAAACDBApPszf6ki9VNAWpAsU6VY1iM59qe5HyE08zr8tw0Sg
+AAAEAVfiT1ThKA3424TcV06ftOd1agt6trEQfYrI6wfULvCsECk+zN/qSL1U0BakCxTpVj
+WIzn2p7kfITTzOvy3DRKAAAAF3RhbWV5ZXJAdGFtLWNhbm9uY2lhbC0xAQIDBAUG
+-----END OPENSSH PRIVATE KEY-----
+"""
+
+TEST_SSH_PUBLIC_KEY = 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMECk+zN/qSL1U0BakCxTpVjWIzn2p7kfITTzOvy3DRK tameyer@tam-canoncial-1'
+
```

Comment:
I don't love having a key in the test code (and lots of thing yell at you about it) even though it doesn't have any value at all. However, `ssh-keygen` isn't available in the GitHub environment, and going through the steps to install it was quite involved. I'm happy to hear suggestions about alternatives (I do think we want a separate one, not to us the user's existing key, for more explicit testing, both in 3 and 4).

Reply:
Let's see if we can use the `openssl` tool for this (eg: `openssl genrsa`).